core/src/main/kotlin/analysis/AnalysisTransformer.kt

```diff
@@ -0,0 +1,11 @@
+package org.jetbrains.dokka.analysis
+
+import org.jetbrains.dokka.EnvironmentAndFacade
+import org.jetbrains.dokka.pages.PlatformData
+
+/**
+ * [AnalysisTransformer] allows to map over
```

Comment:
This description seems incomplete. `to map over...` what is it mapping over?

Reply:
Yes, sure I forgot to commit that change